roles/logging-config/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 - include: logstash.yml
-  when: logging.forward_type != 'filebeat'
+  when: logging.forward_type|default('filebeat') != 'filebeat'
```

Comment:
not sure how would impact other for backward compatibility as other side would need filebeat enabled as well.